src/main/resources/files/BookingController.java.tpl

```diff
@@ -27,7 +27,6 @@ public class BookingController {
                     content = @Content(
                             mediaType = MediaType.APPLICATION_JSON,
                             schema = @Schema(
-                                    ref = "Booking",
```

Comment:
Without the ref value, the OpenAPI document inlines the definition of Booking and no longer reference it.  This change is not wanted!